packages/builder/src/pages/builder/portal/users/groups/[groupId].svelte

```diff
@@ -75,7 +75,8 @@
   let loaded = false
   let editModal, deleteModal
 
-  const scimEnabled = $features.isScimEnabled
+  let scimEnabled = false
+  $: scimEnabled = $features.isScimEnabled
```

Comment:
It would work without the initialisation, but not sure why some variables are initialised and some are not. Is there any logic we should be aware? @PClmnt

Reply:
I think the store usually holds the default value. Would that work here instead?

Reply:
Yes, you are right that the store has already the default value set. Better that way, so we don't have different default values 👍🏽